bench.yaml

```diff
@@ -0,0 +1,35 @@
+# useful for testing absvc.
```

Comment:
oooo this seems cool

Reply:
haha oops this is cool but for testing absvc

maybe for a follow up PR :D